src/env.h

```diff
@@ -528,10 +528,7 @@ class Environment {
   inline Environment(IsolateData* isolate_data, v8::Local<v8::Context> context);
   inline ~Environment();
 
-  void Start(int argc,
-             const char* const* argv,
-             int exec_argc,
-             const char* const* exec_argv,
+  void Start(const ProcessArguments& process_arguments,
```

Comment:
I believe an ABI-compatible wrapper should be added for `Environment::Start()` too, shouldn't it?

Reply:
I'm not sure, but I think this one is private.

Reply:
Oh, yes, you are right. I thought `node.h` includes `env.h`, but it's actually internal. `node.h` just contains a `class Environment;` forward declaration.